src/main/java/hudson/plugins/sidebar_link/ProjectLinks.java

```diff
@@ -56,10 +56,8 @@ public ProjectLinks(List<LinkAction> links) {
     public List<LinkAction> getLinks() { return links; }
 
     @Override
-    public Collection<? extends Action> getJobActions(AbstractProject<?,?> job) {
```

Comment:
There may be other JobProperty methods, which will be changed due to the generic Type change. So it's not enough. 

Probably it makes sense to just spin `2.0` and break the compatibility here. There will be no plugins affected, and I'd guess 99% of System Groovy scripts will feel well even if they use this class (I do it in my pet project for example, but the change is fine for my scripts). WDYT @lvotypko ?